content/articles/oauth-applications.md

```diff
@@ -49,7 +49,7 @@ Adding an application description is optional.
 ![Application Details form](/files/app-details.png)
 
 When created, you'll be taken to the page with your new application, showing your Client ID and Client Secret.
-![App created](files/app-created.png)
+![App created](/files/app-created.png)
```

Comment:
Not for this PR, but since we had 2 images with wrong paths, we could probably write an automated test to verify the correctness of image paths